diff --git a/node_modules/snowpack/lib/index.js b/node_modules/snowpack/lib/index.js
index 2bbb024..8223764 100644
--- a/node_modules/snowpack/lib/index.js
+++ b/node_modules/snowpack/lib/index.js
@@ -44118,6 +44118,7 @@ ${colors.dim(`  At least one dependency must have an ESM "module" entrypoint. Yo
         external: (id) => external.some((packageName) => isImportOfPackage(id, packageName)),
         treeshake: { moduleSideEffects: true },
         plugins: [
+            ...(userDefinedRollup.userPlugins || []),
             rollupPluginAlias_1.rollupPluginAlias({
                 entries: [
                     // Apply all aliases
@@ -54335,6 +54336,7 @@ async function installPackages({ config, isDev, isSSR, installOptions, installTa
         ...installOptions,
         stats: false,
         rollup: {
+            ...(installOptions.rollup || {}),
             plugins: [
                 {
                     name: 'esinstall:snowpack',
@@ -54723,6 +54725,13 @@ exports.default = {
                 if (config.packageOptions.namedExports !== undefined) {
                     installOptions.namedExports = config.packageOptions.namedExports;
                 }
+                if (config.packageOptions.rollup !== undefined) {
+                    installOptions.rollup = {
+                        ...config.packageOptions.rollup,
+                        userPlugins: config.packageOptions.rollup.plugins || [],
+                        plugins: []
+                    };
+                }
             }
             const installResult = await localInstall.installPackages({
                 config,
